packages/ui/src/theme/tokens/components/collection.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  BackgroundColorValue,
+  ColorValue,
+  DesignToken,
+} from '../types/designToken';
+
+interface PaginationTokens {
+  current: {
+    color: DesignToken<ColorValue>;
+    backgroundColor: DesignToken<BackgroundColorValue>;
+  };
+  button: {
+    color: DesignToken<ColorValue>;
+    _hover: {
+      backgroundColor: DesignToken<BackgroundColorValue>;
+      color: DesignToken<ColorValue>;
+    };
+    _disabled: {
+      color: DesignToken<ColorValue>;
+    };
+  };
+}
+
+interface StateTokens {
+  color: DesignToken<ColorValue>;
+  backgroundColor: DesignToken<BackgroundColorValue>;
+}
+
+export interface SearchTokens {
+  color: DesignToken<ColorValue>;
```

Comment:
It is hard to see what this token would be applied to. The input? The label? The wrapper?

Reply:
This token has been renamed to add more clarity